.travis.yml

```diff
@@ -37,11 +37,11 @@ before_script:
 script:
   - composer lint
   - php phpDocumentor.phar -d src -t docs/api --template="checkstyle" # Different than `composer docs` for CI build.
-  - composer unit
+  - ./vendor/bin/phpunit --coverage-clover ./clover.xml tests
```

Comment:
I don't know if it matters, but we're using Symfony's `simple-phpunit` with `composer test` (`composer unit` specifically). Indeed we weren't generating the clover.xml file which obviously is needed!

Reply:
Oh, good point. Fixed.